packages/oruga/src/components/table/Table.vue

```diff
@@ -170,7 +170,7 @@
                             <o-checkbox
                                 :disabled="!isRowCheckable(row)"
                                 :value="isRowChecked(row)"
-                                @click.native.prevent.stop="checkRow(row, index, $event)"
+                                @input="checkRow(row, index, $event)"
```

Comment:
It's correct but we have to stop propagation otherwise the component will emit row click event